critiquebrainz/frontend/external/musicbrainz_db/entities.py

```diff
@@ -0,0 +1,46 @@
+from critiquebrainz.frontend.external.musicbrainz_db.release_group import fetch_multiple_release_groups, get_release_group_by_id
+from critiquebrainz.frontend.external.musicbrainz_db.place import fetch_multiple_places, get_place_by_id
+from critiquebrainz.frontend.external.musicbrainz_db.event import fetch_multiple_events, get_event_by_id
+
+
+def get_multiple_entities(entities):
+    """Fetch multiple entities using their mbids.
+
+    Args:
+        entites: List of tuples containing the entity_id and the entity_type.
```

Comment:
Why `entity_id` and `entity_type` and not simply `entity ID` and `entity type`? It would make sense to, for example, reference dictionary keys exactly by name, but in this case I don't see any point.